File: DarkOrbit 10.0/Net/netty/commands/PetGearTypeModule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ow.Utils;

namespace Ow.Net.netty.commands
{
    class PetGearTypeModule
    {
        public static short ID = 18943;

        public const short BEHAVIOR = 0;
        public const short PASSIVE = 1;
        public const short GUARD = 2;
        public const short GEAR = 3;
        public const short AUTO_LOOT = 4;
        public const short AUTO_RESOURCE_COLLECTION = 5;
        public const short ENEMY_LOCATOR = 6;
        public const short RESOURCE_LOCATOR = 7;
        public const short TRADE_POD = 8;
        public const short REPAIR_PET = 9;
        public const short KAMIKAZE = 10;
        public const short COMBO_SHIP_REPAIR = 11;
        public const short COMBO_GUARD = 12;
        public const short ADMIN = 13;

        public short typeValue = 0;

        public PetGearTypeModule(short typeValue)
        {
            this.typeValue = typeValue;
        }

        public byte[] write()
        {
            var param1 = new ByteArray(ID);
            param1.writeShort(-13816);
            param1.writeShort(-7374);
            param1.writeShort(this.typeValue);
            return param1.Message.ToArray();
        }
    }
}

File: DarkOrbit 9.0/DarkOrbit 9.0 Emulator/src/simulator/netty/serverCommands/PetGearTypeModule.java
package simulator.netty.serverCommands;

import java.io.DataOutputStream;
import java.io.IOException;

import simulator.netty.ServerCommand;

/**
Created by LEJYONER on 21/09/2017.
*/

public class PetGearTypeModule
        implements ServerCommand {

    public static final short BEHAVIOR = 0;    
    public static final short PASSIVE = 1;   
    public static final short GUARD = 2;   
    public static final short GEAR = 3;   
    public static final short AUTO_LOOT = 4;   
    public static final short AUTO_RESOURCE_COLLECTION = 5;  
    public static final short ENEMY_LOCATOR = 6;   
    public static final short RESOURCE_LOCATOR = 7;   
    public static final short TRADE_POD = 8;   
    public static final short REPAIR_PET = 9;   
    public static final short KAMIKAZE = 10;  
    public static final short COMBO_SHIP_REPAIR = 11;   
    public static final short COMBO_GUARD = 12;   
    public static final short ADMIN = 13;
   
    public static int ID = 25441;

    public short typeValue = 0;
    
    public PetGearTypeModule(short param1) {
    	this.typeValue = param1;
    }

    public void write(DataOutputStream param1) {
        try {
            param1.writeShort(ID);
            this.writeInternal(param1);
        } catch (IOException e) {
        }
    }

    protected void writeInternal(DataOutputStream param1) {
        try {
            param1.writeShort(typeValue);
        } catch (IOException e) {
        }
    }
}